railgun/apis/configuration/v1/configsource_types.go

```diff
@@ -0,0 +1,59 @@
+package v1
```

Comment:
todo: these don't need to be API types, remove from KB

Reply:
they do need to have clientset types, which should automatically generate by reference

Reply:
fixed in rebase